.gitlab-ci.yml

```diff
@@ -10,10 +10,12 @@ variables:
   DATADOG_AGENT_BUILDIMAGES: v13321514-644fed4
   S3_CP_CMD: aws s3 cp --only-show-errors --region us-east-1 --sse AES256
   TEST_INFRA_DEFINITIONS_BUILDIMAGES: cb9e86993fd5
+  DEFAUT_MAJOR_VERSION: 7
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
  DEFAULT_MAJOR_VERSION: 7
```

Reply:
oh my :worried: 
Thanks for spotting, fixed